src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue29297.cs

```diff
@@ -0,0 +1,27 @@
+#if !ANDROID
+//Skipping the test case to run in android because the loaded event is not triggered when navigating back to previous page.
```

Comment:
Could you create a new bug and include the link to it here?

Reply:
I have logged new bug for this issue. 
https://github.com/dotnet/maui/issues/29414